test/factories/benchmark.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :benchmark, class: Xccdf::Benchmark do
+    title { Faker::Lorem.word }
+    description { Faker::Lorem.paragraph }
+    ref_id { "#{SecureRandom.uuid}_RHEL-#{os_major_version}" }
+    version { 3.times.map { SecureRandom.rand(10) }.join('.') }
+
+    transient do
+      os_major_version { '7' }
```

Comment:
We can, but please let's do it in a followup. It would need some additional test changes and multiple runs as it would probably introduce sporiadic failures.